lizard_auth_client/tests.py

```diff
@@ -1014,6 +1015,23 @@ def test_change_user_form(self):
         self.assertTrue('last_name' in form.fields.keys())
 
 
+class MockResponse:
+    """
+    Mocks the requests.Response object
+    """
+    def __init__(self, status_code, json_data):
+        self.status_code = status_code
+        self.json_data = json_data
+
+    def json(self):
+        return self.json_data
+
+    def raise_for_status(self):
+        # Raise HTTPError for 4xx and 5xx
+        if self.status_code / 100 in (4, 5):
```

Comment:
Dit gaat niet in python3 werken. Dan krijg je `4.04` in plaats van `4` bij een `404`